components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/role/edit-users.jsp

```diff
@@ -404,40 +404,42 @@
                                                             disPlayName = userName;
                                                         }
                                                         if (users[i].getItemName()
-                                                                    .equals(CarbonConstants.REGISTRY_ANONNYMOUS_USERNAME)) {
+                                                                .equals(CarbonConstants.REGISTRY_ANONNYMOUS_USERNAME)) {
                                                             continue;
                                                         } else if (readOnlyRole && !users[i].getEditable()) {
                                                             doEdit = "disabled=\"disabled\"";
                                                         } else if (session.getAttribute("checkedUsersMap") != null &&
-                                                                   ((Map<String, Boolean>) session
-                                                                           .getAttribute("checkedUsersMap"))
-                                                                           .get(users[i].getItemName()) != null &&
-                                                                   ((Map<String, Boolean>) session
-                                                                           .getAttribute("checkedUsersMap"))
-                                                                           .get(users[i].getItemName()) == true) {
+                                                                ((Map<String, Boolean>) session
+                                                                        .getAttribute("checkedUsersMap"))
+                                                                        .get(users[i].getItemName()) != null &&
+                                                                ((Map<String, Boolean>) session
+                                                                        .getAttribute("checkedUsersMap"))
+                                                                        .get(users[i].getItemName()) == true) {
```

Comment:
Can remove null check and remove == true and just use Boolean.parseBoolean() inside if condition.